src/main/java/io/cdap/plugin/salesforce/plugin/sink/batch/ErrorHandling.java

```diff
@@ -25,7 +25,7 @@ public enum ErrorHandling {
 
   SKIP("Skip on error"),
 
-  STOP("Stop on error");
+  FAIL("Fail on error");
```

Comment:
Where is this being used? If there are existing references, they should reflect the change.

Reply:
ErrorHandling.STOP is not used anywhere in the code. The callers are rather using the methd ErrorHandling.fromValue(string) method call instead. The value in the widget json("Fail on error") does not match with the Enum string ("Stop on error") which currently cause the validate operation to fail.